common/utils/works.js

```diff
@@ -23,7 +23,19 @@ export function getProductionDates(work: Work) {
     .reduce((a, b) => a.concat(b), []);
 }
 
-export function getIiifPresentationLocation(work: Work) {
+export type iiifPresentationLocation = {|
+  locationType: {
+    id: 'iiif-presentation',
+    label: 'IIIF Presentation API',
+    type: 'LocationType',
+  },
+  url: string,
+  type: 'DigitalLocation',
+|};
```

Comment:
Could we start building out the Work type with this?
Naming convention would be `IIIFPresentationLocation`.